### PR TITLE
Create an API for uMap with DRF

### PR DESCRIPTION
**Pull request by davidbgk**
Refs #1333

### CODE REVIEW

**Comment by davidbgk** [comment]
This is really a first step for maps' creation only and not all checks are performed for now (see #TODOs) but it gives a direction.

**Comment by davidbgk** [comment]
At some point, we might want to generate the documentation too: https://www.django-rest-framework.org/topics/documenting-your-api/

File: umap/settings/base.py
Comment (yohanboniface) [inline umap/settings/base.py:1]: Maybe this should pass in master directly ?

**Comment by yohanboniface** [comment]
Cool! Sounds like the way to go

**Comment by davidbgk** [comment]
Closed in favour of https://github.com/umap-project/umap/pull/1396

**Pull request closed without merging**